pkg/ocm/versions.go

```diff
@@ -43,6 +43,7 @@ const (
 
 	MinVersionForAdditionalComputeSecurityGroupIdsDay1 = "4.14.0-0.a"
 	MinVersionForAdditionalComputeSecurityGroupIdsDay2 = "4.11.0-0.a"
+	// TODO: Add new minimal versions to the `features.go` package.
```

Comment:
@oriAdler How about we create a section in `CONTRIBUTING.md` around version-gating a feature rather than a TODO in the code? We could then link to the section in `CONTRIBUTING.md` from here.

Ideally we don't want versions in the CLI at all, but for now we can at least try and be consistent. If we update `CONTRIBUTING.md` and then include an announcement in our Slack channel so that everyone on the team is aware of this new approach?